tools/internal_ci/linux/grpc_e2e_performance_gke.sh

```diff
@@ -19,6 +19,20 @@ cd "$(dirname "$0")/../../.."
 
 source tools/internal_ci/helper_scripts/prepare_build_linux_rc
 
+
+# Environment variables to select repos and branches for various repos.
+# You can edit these lines if you want to run from a fork.
+GRPC_CORE_REPO=grpc/grpc
```

Comment:
If you want devs to be able to customize this script more easily without having to edit it, you can let the shell's environment variables override the script variables with something like `GRPC_CORE_REPO="${GRPC_CORE_REPO:-grpc/grpc}"`

Then you can invoke it using `GRPC_CORE_REPO=drfloob/grpc grpc_e2e_performance_gke.sh`

Reply:
Thank you. That requires other changes too, because the script right now assumes it is invoked by kokoro, not from the command line (it looks for variables like `KOKORO_GITHUB_COMMIT_URL` and others).